pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -448,16 +449,15 @@ func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.R
 		Version: "v1",
 		Kind:    "Node",
 	})
-	err = r.Client.Get(context.Background(), key, nodeCfg)
-	if err != nil {
+	if err := r.Client.Get(ctx, key, nodeCfg); err != nil {
```

Comment:
this style change (our style is not very consistent anyway) should go in its own commit